README.md

```diff
@@ -2,21 +2,20 @@
 
 ## Main features
 
-### `mender-qa` Pipeline run
-
-Set `WATCH_REPOS_PIPELINE` for the list of repositories for which to run
-`mender-qa` Pipeline. See source code for defaults.
-
-This is a Mender specific feature.
-
 ### GitHub -> GitLab sync
 
-All repositories from the configured GitHub organization are synced with GitLab.
+By default all repositories from the configured GitHub organization are synced with GitLab if `SYNC_REPOS_LIST` (comma separated values) is not set.
```

Comment:
I suggest to be here more explicit in what the variable actually does.

For example:
```suggestion
By default all repositories from the configured GitHub organization are synced with GitLab. To select a subset of repositories to sync, set `SYNC_REPOS_LIST` env variable with a comma separated list of repositories.
```

The issue I see with the current text is that it really doesn't say what the variable does, only that it won't sync all repos -> but what repos then will be synced?